Decisions.Box/Data/FolderItem.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using DecisionsFramework.Design.ConfigurationStorage.Attributes;
+
+namespace Decisions.Box
+{
+    [DataContract]
+    [Writable]
+    public class FolderItem
+    {
+        [DataMember]
+        [WritableValue]
+        public string type { get; set; }
+        [DataMember]
+        [WritableValue]
+        public string id { get; set; }
+        [DataMember]
+        [WritableValue]
+        public string sequence_id { get; set; }
+        [DataMember]
+        [WritableValue]
+        public string etag { get; set; }
```

Comment:
Is it an `etag` or a `tag`

Reply:
etag is correct. I'm not sure what it's used for, but it exists from the original implementation also.